aws/sqs.go

```diff
@@ -0,0 +1,79 @@
+package aws
+
+import (
+	"strconv"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+// Returns a formatted string of SQS Queue URLs
+func getAllSqsQueue(session *session.Session, region string, excludeAfter time.Time) ([]*string, error) {
+	svc := sqs.New(session)
+	result, err := svc.ListQueues(&sqs.ListQueuesInput{})
```

Comment:
Is this a paginated API?

Reply:
Bump on this item. If this is paginated, we are only fetching the first page right now...

Reply:
Pagination implemented 😅 .. I'm getting 10 queues per page and creating 20 on tests to, to validate that We'll iterate over the two pages.